cfgov/jinja2/v1/about-us/the-bureau/bureau-structure/index.html

```diff
@@ -46,7 +46,7 @@
 
 {% block content_main %}
     <h1>Bureau Structure</h1>
-    <p>Org chart last updated: August 3, 2016</p>
+    <p>Org chart last updated: July 26, 2016</p>
     <div class="org-chart">
```

Comment:
@sebworks thanks. Looks the png still needs to be updated though -- AJ uploaded one that has the July 26th update date on it.